src/main/java/nl/tudelft/model/wall/MovingWall.java

```diff
@@ -0,0 +1,42 @@
+package nl.tudelft.model.wall;
+
+import nl.tudelft.semgroup4.Modifiable;
+
+import org.newdawn.slick.Image;
+import org.newdawn.slick.SlickException;
+
+public abstract class MovingWall extends AbstractWall {
+    
+    private float speed;
+
+    /**
+     * Creates a moving wall, which will move in a vertical
```

Comment:
The MovingWall does not move vertical / horizontal yet. Could you change the JavaDoc to reflect this?

< documentation